.github/workflows/gradle7.yml

```diff
@@ -0,0 +1,20 @@
+name: Build with Gradle 7.x
+on: pull_request
+jobs:
+  gradle:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Build plugin using hub gradle version
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.4 # https://github.com/runelite/plugin-hub/blob/master/package/gradle/wrapper/gradle-wrapper.properties
+          arguments: shadowJar -x test # tests are already executed by primary gradle task
```

Comment:
I think we should just let tests run, use some of githubs cpu time

Reply:
Plugin hub action doesn't run our tests so I think this would just be an annoyance due to having to retry the action when a flaky test fails